api/app/signals/apps/email_integrations/utils.py

```diff
@@ -126,6 +127,8 @@ def make_email_context(signal: Signal, additional_context: Optional[dict] = None
         'status_state': signal.status.state,
         'handling_message': signal.category_assignment.stored_handling_message,
         'ORGANIZATION_NAME': settings.ORGANIZATION_NAME,
+        'main_category_name': category.parent.name if category.parent else '',
+        'sub_category_name': category.name,
```

Comment:
Should we not use the `public_name` here and have the `name` as a fallback?

Reply:
I think not, as this is what is used currently in mails sent out by Den Bosch. I will expose both.

Reply:
default is now to use public names if available and fall back to the non-public ones if not